packages/datadog-plugin-aws-sdk/src/services/bedrockruntime.js

```diff
@@ -0,0 +1,254 @@
+'use strict'
+
+const BaseAwsSdkPlugin = require('../base')
+const log = require('../../../dd-trace/src/log')
+
+const AI21 = 'AI21'
+const AMAZON = 'AMAZON'
+const ANTHROPIC = 'ANTHROPIC'
+const COHERE = 'COHERE'
+const META = 'META'
+const STABILITY = 'STABILITY'
+const MISTRAL = 'MISTRAL'
```

Comment:
It can be a code smell if there's a bunch of constants with the value being exactly the same as the variable name. It looks like these are something called "providers"? If so maybe either prefix them with `PROVIDER_` or toss them together like this:

```js
const PROVIDER = {
  AI21: 'AI21',
  // ...
};
// ...
if (provider === PROVIDER.AI21) {
```

This way it's super clear the purpose of the otherwise unrelated top level constants.